02-electric/electric-section-one/electric-section-one.css

```diff
@@ -1,14 +1,28 @@
 .electric-section-one-container {
   padding: var(--container-padding-mobile);
+  display: grid;
+}
+
+@media (min-width:500px) {
+  .electric-section-one-container {
+    padding: var(--container-padding-desktop);
+  }
 }
 
 .electric-section-one-subcontainer {
   display: grid;
   grid-template-columns: 1fr;
   justify-content: center;
-  row-gap: var(--items-gap-mobile);
+  row-gap: 30px;
```

Comment:
need to think of maybe a global variable here.. especially if this repeats across many components..

Reply:
i will use this in all the electric page 4 times, its good to have it as global, but i preferred to not touch global style, in this branch.